clients/client-python/tests/integration/test_gvfs_with_gcs.py

```diff
@@ -129,9 +131,17 @@ def _init_test_entities(cls):
             properties=cls.fileset_properties,
         )
 
-        os.environ["GOOGLE_APPLICATION_CREDENTIALS"] = cls.key_file
-        arrow_gcs_fs = GcsFileSystem()
-        cls.fs = ArrowFSWrapper(arrow_gcs_fs)
+        cls.fs = GCSFileSystem(token=cls.key_file)
+
+    # Object storage like GCS does not support making directory and can only create
```

Comment:
I saw gcsfs / s3fs supports `mkdir` according to the doc, maybe you can test them in their expected situations: https://gcsfs.readthedocs.io/en/latest/api.html#gcsfs.core.GCSFileSystem.mkdir, https://s3fs.readthedocs.io/en/latest/api.html#s3fs.core.S3FileSystem.mkdir.

Reply:
The API does support `mkdir` but it takes no effect and the `directory` will not be created for S3 and GCS.

Reply:
Ok, let me check the directory will not be created.